src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentServiceSpec.scala

```diff
@@ -210,4 +209,90 @@ class ConsignmentServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers
 
     parentFolderResult shouldBe parentFolder
   }
+
+  "getSeriesOfConsignment" should "return the series for a given consignment" in {
+    val consignmentRepoMock = mock[ConsignmentRepository]
+    val fileMetadataRepositoryMock = mock[FileMetadataRepository]
+    val fileRepositoryMock = mock[FileRepository]
+    val ffidMetadataRepositoryMock = mock[FFIDMetadataRepository]
+    val fixedUuidSource = new FixedUUIDSource()
+
+    val service: ConsignmentService = new ConsignmentService(consignmentRepoMock,
+      fileMetadataRepositoryMock,
+      fileRepositoryMock,
+      ffidMetadataRepositoryMock,
+      FixedTimeSource,
+      fixedUuidSource)
+
+    val consignmentId = UUID.fromString("d8383f9f-c277-49dc-b082-f6e266a39618")
+    val seriesId = UUID.fromString("bf55455e-2017-11eb-adc1-0242ac120002")
+    val userId = UUID.fromString("ceb8831c-2017-11eb-adc1-0242ac120002")
+    val datetime = Timestamp.from(FixedTimeSource.now)
+    val parentFolder = Option("Parent_folder")
+    val bodyId = UUID.fromString("8eae8ed8-201c-11eb-adc1-0242ac120002")
+    val seriesName = Option("Mock series")
+    val seriesCode = Option("Mock series")
+    val seriesDescription = Option("Series description")
+    val bodyName = Option("Mock department")
+    val bodyCode = Option("Mock department")
+    val bodyDescription = Option("Body description")
```

Comment:
What do you think about extracting out some methods which create the consignment, series and body, to remove some of the lengthy setup from the test itself? They could provide default values for the things that we don't care about in this test, such as the body description or the consignment creation time.

Reply:
Ah yes, this is a good idea; it was something I was thinking about too.